apps/admin-gui/src/app/vos/pages/member-detail-page/member-overview/member-overview.component.ts

```diff
@@ -224,4 +225,32 @@ export class MemberOverviewComponent implements OnInit {
       () => (this.loading = false)
     );
   }
+
+  private sponsorThisMember(): void {
+    const config = getDefaultDialogConfig();
+    config.width = '400px';
+    config.data = {
+      theme: 'member-theme',
+      member: this.member,
+      sponsors: this.sponsors,
+    };
+
+    const dialogRef = this.dialog.open(SponsorThisMemberDialogComponent, config);
+
+    dialogRef.afterClosed().subscribe((edited) => {
+      if (edited) {
+        this.loading = true;
+        this.membersService.getRichMemberWithAttributes(this.member.id).subscribe((member) => {
+          this.member = member;
+          if (this.member.sponsored) {
+            this.usersManager.getSponsorsForMember(this.member.id, null).subscribe((sponsors) => {
```

Comment:
Here we are sure the member is sponsored (result of the dialog actions) and the check can be omitted.